protos/frame-pointer-validator.proto

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+syntax = "proto3";
+
+message ModuleInformation {
+  string module_path = 1;
+  repeated FunctionInformation functions = 2;
+}
+
+message FunctionInformation {
+  uint64 offset = 1;
+  uint64 size = 2;
+}
```

Comment:
you have this and also there is FunctionInfo struct in the code. Is it necessary to have both? can't you use just this one?

Reply:
I wanted to avoid using the FunctionInfo struct, which gets generated by the proto.

Reply:
Unless we have good reasons for doing so, we shouldn't duplicate protos as structs. (A good reason would be a measurable performance difference.)

Reply:
The `FunctionInfo` struct is used in `OrbitCore/FramePointerValidator` and OrbitCore is not linked against OrbitProtos. Thus, I would either need to link them, or stick with the struct.

Reply:
You can link it against proto.

Reply:
done. thanks.